core/views.py

```diff
@@ -205,81 +206,36 @@ class AboutView(SetEtagMixin, TemplateView):
     template_name = 'core/about.html'
 
 
-class PrivacyCookiesDomesticCMS(TemplateView):
-    template_name = 'core/privacy_cookies-domestic-cms.html'
-
-    def get_context_data(self, *args, **kwargs):
-        response = cms_client.lookup_by_slug(
-            slug=EXPORT_READINESS_PRIVACY_AND_COOKIES_SLUG,
-            language_code=translation.get_language(),
-            draft_token=self.request.GET.get('draft_token'),
-        )
-        return super().get_context_data(
-            page=handle_cms_response(response),
-            *args, **kwargs
-        )
+class PrivacyCookiesDomesticCMS(GetCMSPageMixin, TemplateView):
+    template_name = 'core/info_page.html'
+    slug = EXPORT_READINESS_PRIVACY_AND_COOKIES_SLUG
 
 
 class PrivacyCookiesInternationalCMS(PrivacyCookiesDomesticCMS):
-    template_name = 'core/privacy_cookies-international-cms.html'
+    template_name = 'core/info_page_international.html'
```

Comment:
is the name of the page changing? Maybe renaming the view would make sense then, it's nice when template filenames are similar to the views that use them.

Reply:
name of the page isn't changing so the view name can stay the same.`info_page_international` template is used by both the T&Cs and cookies pages. several pages now use a similar template (cookies, T&Cs, dashboard notes) so this is just to maximise code reuse

Reply:
ah, nice!